src/builder.rs

```diff
@@ -290,6 +295,15 @@ in {}
 
         let info = run(&::NixFile::from(cas.file_from_string(&nix_drv)?), &cas).unwrap();
         assert!(info.exec_result.success());
+        assert!(info
+            .named_drvs
+            .get("shell")
+            .unwrap()
+            .to_string_lossy()
+            .contains("-shell.drv"));
```

Comment:
I’d pull that out into a different test (if we really want/need to test it, the code itself (after #117) is going to have the asserts and will fail if it’s missing).